### ISSUE TITLE
can not run script

### ISSUE THREAD

**Issue by Niefee** [rgbkrk/atom-script#1450]
environment:
OS: windows10 64bit
atom: 1.21.0
atom-script: 3.17.1
try to run: node(js)  the script

```js
var aa = new Date();
console.log(aa);
```

but the result is always show : 

```js
'bash' �����ڲ����ⲿ���Ҳ���ǿ����еĳ���
���������ļ���
[Finished in 0.138s]
```

**Comment by danielbayley** [comment]
Duplicate of [@Niefee see] #1449. Fixed in #1451.

**Comment by Niefee** [comment]
thinks
